pkg/apis/scope/v0alpha1/types.go

```diff
@@ -0,0 +1,26 @@
+package v0alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+type Scope struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec ScopeSpec `json:"spec,omitempty"`
+}
+
+type ScopeSpec struct {
+	// S does nothing and is a stub
+	S string `json:"s"`
```

Comment:
🤣 yup -- this is my biggest question... what goes here

Reply:
Let me get back to you soon! :)